custom/ewsghana/api.py

```diff
@@ -313,6 +313,7 @@ def _set_up_supply_point(self, location, ews_location):
             supply_point_with_stock_data = filter(
                 lambda x: x.last_reported and x.active, ews_location.supply_points
             )
+            location.save()
```

Comment:
I moved save to _set_up_supply_point because in main location_sync method we don't know which site_code should be used(for facility we are using code from supply point).